build_extra/rust/rust.gni

```diff
@@ -229,7 +213,12 @@ template("rust_crate") {
                                    "crate_name",
                                    "crate_version",
                                  ])
-          crate_name_and_version = "$crate_name-$crate_version"
+          crate_suffix = exec_script("//tools/sha256sum.py",
```

Comment:
`exec_script` is synchronous and slows down parallelization of builds. Quoting [Chromium's .gn file](https://chromium.googlesource.com/chromium/src/+/master/.gn):

> These are the list of GN files that run exec_script. This whitelist exists to force additional review for new uses of exec_script, which is strongly discouraged.

> You should almost never need to add new exec_script calls. exec_script is slow, especially on Windows, and can cause confusing effects. Although individually each call isn't slow or necessarily very confusing, at the scale of our repo things get out of hand quickly. By strongly pushing back on all additions, we keep the build fast and clean.

Not saying you need to change it - just relaying these docs.

Reply:
I am aware of that. There is only one crate that requires this treatment atm, but I agree it is suboptimal to do this.